### PR TITLE
Remove code sample from Geocoding in Rails Apps

### PR DESCRIPTION
**Pull request by malandrina**

### CODE REVIEW

**Comment by malandrina** [comment]
This commit removes the `Location` code sample and discussion around the code sample. I'd like the 'Geocoding in Rails Applications' part to be a more general overview of geocoding and how you do it in Rails. I think it makes sense to introduce and discuss the tactic of only geocoding when necessary in the performance improvement section. Thoughts?

File: book/strategies/geocoding_in_rails_applications.md [hunk 1]
```diff
@@ -1,58 +1,21 @@
...
-ensures we don't spend precious time geocoding the record unless it is ready
-to be persisted.
-
-Once the record has coordinates assigned, it should be queryable. To find all
-locations within twenty miles of Boston:
-
-```ruby
-Location.near('Boston', 20)
-```
+There are [plenty of gems](#gems) which can be used to automate the process of
```
Comment (joshuaclayton) [inline book/strategies/geocoding_in_rails_applications.md]: "which can be used to automate" => "which automates"?

File: book/strategies/geocoding_in_rails_applications.md [hunk 2]
```diff
@@ -1,58 +1,21 @@
...
+geocoding data as it's added or updated. The geocoded object often exposes a
+method to be used during the geocoding process; the gem then updates latitude
+and longitude itself once it has calculated the object's position. The gem will
```
Comment (joshuaclayton) [inline book/strategies/geocoding_in_rails_applications.md]: "once it has calculated" => "after calculating"?

**Comment by joshuaclayton** [comment]
@malandrina Giving this a :+1:; I'm glad we're able to pull this out. I think the section discussing the example app makes much more sense for this.

**Pull request merged**